docs/src/main/asciidoc/spring-security.adoc

```diff
@@ -228,8 +228,160 @@ The table below summarizes the supported annotations:
 |@Secured
 |
 
+|@PreAuthorize
+|See next section for more details
+
 |===
 
+==== @PreAuthorize
+
+Quarkus provides support for some of the most used features of Spring Security's `@PreAuthorize` annotation.
+The expressions that are supported are the following:
+
+* hasRole
```

Comment:
I think this whole thing would be better with a description list: https://asciidoctor.org/docs/user-manual/#description-list .

You can group several paragraphs together by having a `+` on the new line.

Reply:
I'll check it out, thanks!

Reply:
I did something that hopefully looks OK :)

Reply:
Well, it did not :).

I took the liberty to add a separate commit so that you see what I meant. Feel free to squash it before merging.

Reply:
Thanks a lot!

I see no reason to squash TBH. If you approve, then we can merge as is 😎